src/flatc.cpp

```diff
@@ -105,6 +124,8 @@ std::string FlatCompiler::GetUsageString(const char *program_name) const {
     "  --raw-binary       Allow binaries without file_indentifier to be read.\n"
     "                     This may crash flatc given a mismatched schema.\n"
     "  --size-prefixed    Input binaries are size prefixed buffers.\n"
+    "  --schema-size-prefixed\n"
```

Comment:
why not reuse the existing flag for this?

Reply:
The reason is that with `flatc --schema --binary --size-prefixed` a size prefixed schema is created.  Now if you want to read or create a plain buffer with a size prefixed schema or a size prefixed buffer with a plain schema you need flags to tell which is which.  I imagine that the size-prefixed schema case itself is rare so you rarely ever need to give this flag.

Reply:
ah, makes sense.